Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -177,23 +179,28 @@ public void initializeTaskList() {
         filterAlleStart();
     }
 
-    private void sortList(Criteria inCrit) {
-        inCrit.addOrder(Order.desc("priority"));
-
-        Order order = Order.asc("proc.title");
+    /**
+     * It is possible that sorting by related object can be hard to achieve. Right
+     * now it is replaced with sorting by id.
+     * 
+     * @return sort clause for query
+     */
+    private String sortList() {
+        String sort = SortBuilders.fieldSort("priority").order(SortOrder.ASC).toString();
+        /* Order order = Order.asc("proc.title"); */
         if (this.sortierung.equals("schrittAsc")) {
-            order = Order.asc("title");
+            sort += ", " + SortBuilders.fieldSort("title").order(SortOrder.ASC).toString();
         }
         if (this.sortierung.equals("schrittDesc")) {
-            order = Order.desc("title");
+            sort += ", " + SortBuilders.fieldSort("title").order(SortOrder.DESC).toString();
         }
         if (this.sortierung.equals("prozessAsc")) {
-            order = Order.asc("proc.title");
+            sort += ", " + SortBuilders.fieldSort("process").order(SortOrder.ASC).toString();
         }
         if (this.sortierung.equals("prozessDesc")) {
-            order = Order.desc("proc.title");
+            sort += ", " + SortBuilders.fieldSort("process").order(SortOrder.DESC).toString();
         }
-        if (this.sortierung.equals("batchAsc")) {
+        /*if (this.sortierung.equals("batchAsc")) {
```

Comment:
If your out-commented block is not used anymore or obsolete, then remove this. If this should be fixed later, then add a comment / TODO annotation.